build/Makefile.kits

```diff
@@ -27,7 +27,16 @@ ifneq ($(COMBINED_PACKAGES),1)
     else
       ifeq ($(PF_DISTRO),SUSE)
         SCX_PREFIX=scx
-        OUTPUT_PACKAGE_PREFIX=$(SCX_PREFIX)-$(SCX_BUILDVERSION_MAJOR).$(SCX_BUILDVERSION_MINOR).$(SCX_BUILDVERSION_PATCH)-$(SCX_BUILDVERSION_BUILDNR).sles.$(PF_MAJOR).$(PF_ARCH)
+        SSLVERSION = `openssl version`
```

Comment:
Check if it possible to create a universal package for SLES 15 PPC

Reply:
will try, thanks.